clusters/app.ci/assets/bugzilla-backporter.yaml

```diff
@@ -0,0 +1,111 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ImageStream
+  apiVersion: image.openshift.io/v1
+  metadata:
+    name: bugzilla-backporter
+    namespace: ci
+  spec:
+    lookupPolicy:
+      local: true
+    tags:
+    - name: latest
+      importPolicy:
+        scheduled: true
+      from:
+        kind: DockerImage
+        name: registry.svc.ci.openshift.org/ci/bugzilla-backporter
+- apiVersion: v1
+  kind: Service
+  metadata:
+    namespace: ci
+    labels:
+      app: prow
+      component: bugzilla-backporter
+    name: bugzilla-backporter
+  spec:
+    ports:
+    - name: main
+      port: 80
```

Comment:
The port for the deployed container is 8080. Does this need to match that?

Reply:
no, the `targetPort` needs to match that and it does, the port named `main` is 8080